Lecture 3/GithubApp/app/src/main/java/mff/mdp/demoapp/MainActivity.kt

```diff
@@ -0,0 +1,71 @@
+package mff.mdp.demoapp
+
+import android.os.Bundle
+import androidx.activity.ComponentActivity
+import androidx.activity.compose.setContent
+import androidx.activity.enableEdgeToEdge
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material3.ExperimentalMaterial3Api
+import androidx.compose.material3.Scaffold
+import androidx.compose.material3.Text
+import androidx.compose.material3.TopAppBar
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.lifecycle.viewmodel.compose.viewModel
+import androidx.navigation.compose.NavHost
+import androidx.navigation.compose.composable
+import androidx.navigation.compose.rememberNavController
+import androidx.navigation.toRoute
+import kotlinx.serialization.Serializable
+import mff.mdp.demoapp.ui.HomeScreen
+import mff.mdp.demoapp.ui.ProfileScreen
+import mff.mdp.demoapp.ui.theme.DemoAppTheme
+import mff.mdp.demoapp.viewmodels.ProfileViewModel
+
+class MainActivity : ComponentActivity() {
+
+    @OptIn(ExperimentalMaterial3Api::class)
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        enableEdgeToEdge()
+        setContent {
+            DemoAppTheme {
+                Scaffold(
+                    modifier = Modifier.fillMaxSize(),
+                    topBar = {
+                        TopAppBar(title = { Text(text = getString(R.string.app_name)) })
+                    }
+                ) { innerPadding ->
```

Comment:
For educational purposed, it is maybe better to use `content` as named parameter